docs/docs/60-new-docs/50-user-guide/20-how-to-guides/14-working-with-stages.md

```diff
@@ -484,11 +484,35 @@ Users with credentials for and sufficient permissions within the Kargo control p
 
 ### Promoting Freight to a Stage
 
-:::note
-**Maintainers Note**
+<Tabs groupId="promoting-freight-to-a-stage">
+<TabItem value="ui" label="Using the UI" default>
 
-Placeholder
-:::
+1. Click on the target icon to the left of the `Stage`.
+
+    ![Promote Freight to a Stage](img/promote-freight-to-a-stage.png)
+
+2. Select the detected `Freight` from the <Hlt>Freight Timeline</Hlt>, and click <Hlt>Yes</Hlt> to promote:
+
+    ![Promote Freight to a Stage](img/promote-freight-to-a-stage-2.png)
+
+</TabItem>
+
+<TabItem value="cli" label="Using the CLI">
+
+To promote `Freight` to a `Stage` using the CLI, run:
+
+```shell
+kargo promote --stage <stage> --project <project> --freight <freight>
+```
+
+_Alternatively_, you can reference the `Freight` you want to promote using its alias:
+
+```shell
+kargo promote --stage <stage> --project <project> --freight-alias <freight-alias>
+```
```

Comment:
~~~suggestion
To promote `Freight` to a `Stage` using the CLI, run:

```shell
kargo promote \
  --project kargo-demo \
  --freight f5f87aa23c9e97f43eb83dd63768ee41f5ba3766 \
  --stage prod
```

Alternatively, you can reference the `Freight` you wish to promote using its alias:

```shell
kargo promote \
  --project kargo-demo \
  --freight-alias frozen-tauntaun \
  --stage prod
```
~~~